libs/platform/src/lib/components/action-bar/action-bar.component.html

```diff
@@ -2,13 +2,13 @@
     <div fd-action-bar-header>
         <div fd-action-bar-back *ngIf="showBackButton">
             <button
-                class="sap-icon--navigation-left-arrow"
                 fd-button
                 [compact]="compact"
                 [fdType]="'transparent'"
                 [attr.aria-label]="backButtonLabel"
                 data-tag="actionbar__back-button"
                 [attr.title]="backButtonLabel"
+                [glyph]="navigationArrow$ | async"

```

Comment:
What about case where navigation arrow is null so you there is no icon? you want to have this case too.

Reply:
Without back button case is already impleanted with showbackbutton flag in action if this flag is false it won't load back button element.